keystones/tests/mocks.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+
+
+class DiscordUser:
+    """
+    A mock Discord User object.
+    """
+    def __init__(self, user_id=1, name="Discord User"):
+        self.id = user_id
+        self.name = name
+
+
+class DiscordMessage:
+    """
+    A mock Discord Message object.
+    """
+    def __init__(self, content: str, author: DiscordUser or None):
```

Comment:
```suggestion
    def __init__(self, content: str, author=None: DiscordUser):
```
This way would work better because then you don't actually have to pass None when creating a new object

Reply:
Oh yeah lol